terminus-ui/input/src/input.component.spec.ts

```diff
@@ -901,6 +899,17 @@ describe(`TsInputComponent`, function() {
       expect(component.selected.emit).toHaveBeenCalledWith(new Date('01-01-2018'));
     });
 
+    test('should return if target is not set', () => {
+      const fixture = createComponent(TestComponents.DateFilter);
+      const component = fixture.componentInstance.inputComponent;
+      component._valueChange.emit = jest.fn();
+      component.selected.emit = jest.fn();
+      component.onInput(null as any);
+
+      expect(component._valueChange.emit).not.toHaveBeenCalled();
+      expect(component.selected.emit).not.toHaveBeenCalledWith();
```

Comment:
Should this line also be `toHaveBeenCalled()` vs `...With()`?

Reply:
yes. copy/paste error. fixed